File: apps/default/user/vistas/widgets/js/alert.js
$(document).ready(function () {
{{a=["hola mundo"]}}
	{{for elem in ["a","b","c"]:}}
		{{if elem =="a":}}
			alert("{{print elem}}");
		{{else:}}
			{{pass}}
	{{pass}}
	
console.log('javascript');
})

File: apps/default/user/vistas/widgets/js/alert.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
print '''$(document).ready(function () {'''
a=["hola mundo"]
print '''	'''
for elem in ["a","b","c"]:
  print '''		'''
  if elem =="a":
    print '''			alert("'''    +str(elem)    +'''");		'''
  else:
    print '''			'''
    pass
  print '''	'''
  pass
print '''	console.log('javascript');})'''